<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP批量上传工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/qtdemo.ico</normaloff>icon/qtdemo.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="start_upload">
        <property name="text">
         <string>上传</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_upload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="FileList" name="file_list">
          <property name="minimumSize">
           <size>
            <width>232</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DropOnly</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>文件</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>大小</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>修改日期</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎使用FTP批量上传工具。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>上传方案：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="upload_solution"/>
          </item>
          <item>
           <widget class="QToolButton" name="add_solution">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;把当前上传服务器列表添加到上传方案中&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon/0949090.png</normaloff>icon/0949090.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remove_solution">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;删除当前选择的上传方案&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon/0949091.png</normaloff>icon/0949091.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="selected_servers">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>平台</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>服务器</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>路径</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="move2left">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加到上传服务器列表中&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>左移</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icon/09490922.png</normaloff>icon/09490922.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="move2right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从上传服务器列表中删除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>右移</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icon/09490920.png</normaloff>icon/09490920.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>平台：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="platform"/>
          </item>
          <item>
           <widget class="QPushButton" name="refresh_server_list">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>刷 新</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon/09490938.png</normaloff>icon/09490938.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="server_list">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>平台</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>服务器</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>路径</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_operation">
    <property name="title">
     <string>操作(&amp;O)</string>
    </property>
    <addaction name="menu_clear_log"/>
    <addaction name="separator"/>
    <addaction name="menu_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="menu_about"/>
   </widget>
   <addaction name="menu_operation"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_server_manage">
   <property name="text">
    <string>管理服务器</string>
   </property>
  </action>
  <action name="menu_exit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="menu_about">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_add_server">
   <property name="icon">
    <iconset>
     <normaloff>icon/0949090.png</normaloff>icon/0949090.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_edit_server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/09490944.png</normaloff>icon/09490944.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="action_copy_server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/0949092.png</normaloff>icon/0949092.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_delete_server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/0949091.png</normaloff>icon/0949091.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_move2left">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/09490922.png</normaloff>icon/09490922.png</iconset>
   </property>
   <property name="text">
    <string>左移</string>
   </property>
  </action>
  <action name="menu_clear_log">
   <property name="text">
    <string>清空日志(&amp;C)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileList</class>
   <extends>QTableWidget</extends>
   <header>control/filelist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>menu_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
